.github/workflows/build.yml

```diff
@@ -38,11 +38,17 @@ jobs:
         if: "github.ref != 'refs/heads/master'"
 
   build:
-    name: Build with Go ${{ matrix.go }}
-    runs-on: ubuntu-latest
+    name: Build (Go ${{ matrix.go }}, ${{ matrix.os }})
```

Comment:
```suggestion
    name: Build with (Go ${{ matrix.go }}, ${{ matrix.os }})
```

Reply:
If name is too long, it can not be visible in github actions
Thus shorter name is better.